uint8 id
uint8 cmd_mode
float32 torque 
float32 Pos 
float32 speed 
float32 kp 
float32 kd 
